src/NServiceBus.Metrics.ServiceControl/ReportingOptions.cs

```diff
@@ -1,10 +1,13 @@
 ﻿namespace NServiceBus.Metrics.ServiceControl
 {
+    using System;
+
     class ReportingOptions
     {
         internal string ServiceControlMetricsAddress;
         internal string EndpointInstanceIdOverride;
-
+        public TimeSpan TimeToBeReceived { get; set; } = TimeSpan.FromDays(7);
```

Comment:
Should this be configurable? Or is it the default?

Reply:
It's the default and it is configurable via extension method:

```c#
public static void SetServiceControlTTBR(this NServiceBus.BusConfiguration busConfiguration,
   System.TimeSpan timeToBeReceived) { } 
```